apiml-security-common/src/main/java/org/zowe/apiml/security/common/audit/RauditxService.java

```diff
@@ -129,7 +129,7 @@ public void verifyPrivileges() {
                         "FACILITY", "IRR.RAUDITX", "READ"
                     );
```

Comment:
maybe to have a solution in between to not hide the information, can we have an info message here showing the result of the verification? If it's false or not visible we can enable debug logging for troubleshooting